projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loans.component.ts

```diff
@@ -72,12 +77,12 @@ export class PatronProfileLoansComponent implements OnInit, OnDestroy {
     );
     this._subscription.add(
       this._patronProfileMenuService.onChange$.subscribe(() => {
-        this._initialLoad();
+        this.sortCriteria = 'duedate';
+        this.ngOnDestroy();
+        this.ngOnInit();
```

Comment:
Arggh. Is it really a good idea to call life cycle hook function like that ? What's the feeling of @Garfield-fr and/or @jma.